### PR TITLE
Upgrade to support new hugo releases.

### PR DESCRIPTION
**Pull request by otroan**
Hugo from 0.60 upwards is CommonMark compliant.
Apparently that implies that HTML markup is considered unsafe.
Set unsafe=true for now.

After 0.56 symlinks to directories are not supported.
Using module.mounts instead.

Signed-off-by: Ole Troan <ot@cisco.com>

### CODE REVIEW

**Comment by edwarnicke** [comment]
Big +1 on moving to module mounts, but it appears not to be working:

https://deploy-preview-82--fdio.netlify.app/docs/vpp/latest

**Comment by edwarnicke** [comment]
https://deploy-preview-82--fdio.netlify.app/vppproject/vpptech/ 

image


differs from  https://fd.io/vppproject/vpptech/

image

**Comment by edwarnicke** [comment]
https://deploy-preview-82--fdio.netlify.app/community/join/

image


differs from https://fd.io/community/join/

image

File: netlify.toml [hunk 1]
```diff
@@ -3,28 +3,28 @@ publish = "public"
...
 command = "hugo  --gc --minify"
 
 [context.branch-deploy.environment]
-HUGO_VERSION = "0.53"
+HUGO_VERSION = "0.79.0"
 
 [context.next.environment]
-HUGO_ENABLEGITINFO = "true"
\ No newline at end of file
+HUGO_ENABLEGITINFO = "true"
```
Comment (edwarnicke) [inline netlify.toml:30]: Hold for fixes

**Review by edwarnicke — CHANGES_REQUESTED** [review]

File: netlify.toml (hunk 1 above)
Comment (otroan) [reply netlify.toml:30]: I didn't add this line mind you. It was only git pointing out that it was lacking a newline at EOF.

**Review by edwarnicke — APPROVED** [review]

**Pull request merged**